src/foam/nanos/crunch/predicate/CapabilityIsStatus.js

```diff
@@ -43,9 +43,9 @@ foam.CLASS({
     },
     {
       class: 'Boolean',
-      name: 'includeGracePeriod',
+      name: 'includeRenewablePeriod',
```

Comment:
shouldn't this be a separate property? or maybe something like `includeRenewableAndGracePeriod`

Reply:
Yup will change

Reply:
how about inlcudeRenewable as renewalbe seems to be used to mean that a ucj is either in grace period or renewable period or expired?

Reply:
yup sounds good!